scripts/grafana-server/start-server

```diff
@@ -48,7 +48,7 @@ $RUNDIR/bin/"$ARCH"grafana-server \
   --pidfile=$RUNDIR/pid \
   cfg:server.http_port=$PORT \
   cfg:server.router_logging=1 \
-  cfg:app_mode=development \
+  cfg:app_mode=test \
```

Comment:
Would this fit better in `custom.ini`? Also, are there some cases where `development` is required?

Reply:
I don't want the CSP meta tag in index-template to be applied unless it is in testing. If using the development mode, it means that the CSP will be applied in both the meta tag in the HTML and in the HTTP header. It feels like this could cause some weird behavior. Or maybe we can just apply for all modes? What do you think?

Reply:
I am unaware of a "test" mode...  the only modes that exist are development/production.

This PR should not require changes to anything in this folder

Reply:
interesting -- it does exist, but this is the only reference to it.  *maybe* it is set in the CI environments?  

If we need different behavior between produciton+dev, that is totally fine, but lets not introduce a new testing mode for this feature